include/vctr/Containers/VctrBase.h

```diff
@@ -600,6 +600,30 @@ class VctrBase : private Config,
         return it == rend() ? std::nullopt : std::optional<size_t> (std::distance (it, rend()) - 1);
     }
 
+    /** Returns a copy of the first element value which is greater or equal to bound or std::nullopt if no such value is found.
+
+        This function requires that the elements are sorted.
+     */
+    std::optional<value_type> firstValueGreaterThanOrEqualTo (const value_type& bound) const
```

Comment:
so if you change the name of the function from lower bound to the new one, the parameter name "bound" might be misleading.

Reply:
Mhm, what parameter name would you suggest?

Reply:
maybe "valueToFind" or just "value" but "bound" means nothing here :)